examples/render-helm-chart-kustomize-private-oci/password

```diff
@@ -0,0 +1,2 @@
+# This will contain the SA key for acessing the private OCI registry. This file will be populated
+# by the CI, so this test won't work locally.
```

Comment:
is this the secret yaml resource or user/password? I'd suggest give an example to show the required format, but using fake data